refinery/core/utils.py

```diff
@@ -116,19 +116,20 @@ def invalidate_cached_object(instance, is_test=False):
         return mc
 
 
-def get_absolute_url(string):
+def build_absolute_url(string):
```

Comment:
Why the name change?

Reply:
ilya mentioned it in the ticket and it is more descriptive of what the function does.